### ISSUE TITLE
Q: fold enrichment and IP/input ratio

### ISSUE THREAD

**Issue by BioEpi** [macs3-project/MACS#517]
Hello,
    I have run MACS3 to call peaks from IP and input files, and obtained the peaks and related fold enrichment. **Peaks** are filtered based on fold enrichment>5 and q value<1e-5.

I also manually calculated the (IP CPM)/(input CPM) ratio for these **Peaks**. 

In theory,  `the (IP CPM)/(input CPM) ratio` for these **Peaks** should be > 5. However, what I got was that `the (IP CPM)/(input CPM) ratio` < 3.
Thus, I am a little confused what the difference between `the (IP CPM)/(input CPM) ratio` and fold enrichment output by MACS3.

**Comment by taoliu** [comment]
Hi @BioEpi , the 'fold enrichment output' from MACS3 is the fold enrichment of the peak summit at the exact genomic coordinate. This also applies to other output, including p-value and q-value.  It will be different if you manually take the number of reads in the peak from IP and input and then calculate these values again.

**Comment by BioEpi** [comment]
thanks a lot.
fold enrichment===(number in summit of a peak)/(normalized numer in λlocal), right?
for example, number in summit is 100, the normalized numer in λlocal is 20, so the fold enrichment is 5.

Do you think it is a good idea to calculate the ratio via IP/input?

**Comment by taoliu** [comment]
Yes. That's the way we calculate the fold enrichment, p-value and q-value to report in the output file. Basically, MACS computes all these values in basepair resolution after smoothing the pileup.
